### ISSUE TITLE
Jest versions later than 27.x are not working

### ISSUE THREAD

**Issue by thep-dr** [FormidableLabs/react-native-owl#167]
Hi FormidableLabs,

Thanks for a great tool 🥳 

I have trouble running my tests if I upgrade jest to a version later than 27.x. If I upgrade to 28.1.3 for example, the transformation of my tests fail.

I have the following `owl.config.json`:
```
"devDependencies": {
    ...
    "jest": "^28.1.3",
    "react-native-owl": "^1.2.3",
    ...
},
```

I have created a branch for reproduction here: https://github.com/thep-dr/react-native-owl/tree/upgrade-jest

I have tried to play around with different transformer options, but no luck.

The run command fails with the following issue:
```
 FAIL  ./App.owl.tsx
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    /Users/xxx/Documents/xxx/App.owl.tsx:3
    import { takeScreenshot } from 'react-native-owl';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at Runtime.createScriptFromCode (../../jest-runtime/build/index.js:1796:14)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.502 s

```

**Comment by frankcalise** [comment]
Can confirm running into this as well. I downgraded to 

```
"@types/jest": "^27",
"babel-jest": "^27",
"jest": "^27",
```

and the test was able to pass. This was on Expo 49

**Comment by frankcalise** [comment]
I think related to #166 in some way maybe

**Comment by robwalkerco** [comment]
This issue was resolved by #168
